src/coreclr/debug/runtimeinfo/datadescriptor.h

```diff
@@ -757,6 +760,33 @@ CDAC_TYPE_END(CalleeSavedRegisters)
 CDAC_TYPES_END()
 
 CDAC_GLOBALS_BEGIN()
+
+#if defined(TARGET_UNIX)
+CDAC_GLOBAL_STRING(OperatingSystem, unix)
+#elif defined(TARGET_WINDOWS)
+CDAC_GLOBAL_STRING(OperatingSystem, windows)
+#else
+#error Unknown OperatingSystem.
+#endif
```

Comment:
Add an entry to `src/coreclr/debug/runtimeinfo/configure.h.in` using the `CLR_CMAKE_TARGET_OS` define? Unless you specifically want Windows vs non-Windows.